_posts/2018-11-29-craft-consultants.md

```diff
@@ -72,4 +72,6 @@ We aim to have people:
 - get out of comfort zones — by working on different setups (be it a team or a product),
 - becoming Crafters — by doing all of that.
 
+* _Quote redacted for inclusivity purposes._
```

Comment:
Due to the change of the quotation, this does not need to be here. Should even be removed.